candidate-standard/annex_ats.adoc

```diff
@@ -10,7 +10,7 @@ implementations of the EDR API. An ATS provides a basis for developing an Execut
 
 The abstract test cases (assertions) are organized into test groups that correspond to distinct conformance test classes defined in the EDR API specification.
 
-OGC Web APIs are not a Web Services in the traditional sense. Rather, they define the behavior and content of a set of Resources exposed through a Web Application Programing Interface (Web API). Therefore, an API may expose resources in addition to those defined by the standard. A test engine must be able to traverse the API, identify and validate test points, and ignore resource paths which are not to be tested.
+OGC Web APIs are not Web Services in the traditional sense. Rather, they define the behavior and content of a set of Resources exposed through a Web Application Programing Interface (Web API). Therefore, an API may expose resources in addition to those defined by the standard. A test engine must be able to traverse the API, identify and validate test points, and ignore resource paths which are not to be tested.
```

Comment:
OGC Web APIs or OGC APIs?  We see both in the PR.

Reply:
Changing all references to OGC APIs would probably help avoid any confusion